server/src/test/java/org/elasticsearch/cluster/coordination/CoordinatorTests.java

```diff
@@ -883,12 +882,12 @@ public void testIncompatibleDiffResendsFullState() {
         final PublishClusterStateStats prePublishStats = follower.coordinator.stats().getPublishStats();
         logger.info("--> submitting first value to {}", leader);
         leader.submitValue(randomLong());
-        cluster.runFor(DEFAULT_CLUSTER_STATE_UPDATE_DELAY + defaultMillis(PUBLISH_TIMEOUT_SETTING), "publish first state");
+        cluster.runFor(DEFAULT_CLUSTER_STATE_UPDATE_DELAY, "publish first state");
         logger.info("--> healing {}", follower);
         follower.heal();
         logger.info("--> submitting second value to {}", leader);
         leader.submitValue(randomLong());
-        cluster.stabilise(DEFAULT_CLUSTER_STATE_UPDATE_DELAY);
+        cluster.stabilise();
```

Comment:
I think we should have an explicit duration here - we do not expect there to be an election so it should not be necessary to wait for a full stabilisation.

Reply:
Would you like to suggest such a minimal bound? The point of this test is to check the cluster state diffing logic, not determine any kind of upper bound on stabilization after a follower has been unresponsive.

Reply:
I haven't had time to do so, and would rather merge this PR than block on that.